user/views.py

```diff
@@ -78,10 +84,7 @@ def post(self, request, *args, **kwargs):
             serializer.save()
             return Response(status=status.HTTP_201_CREATED)
         else:
-            return Response(
-                data={"message": "Bad request.", "code": status.HTTP_400_BAD_REQUEST},
-                status=status.HTTP_400_BAD_REQUEST,
-            )
+            return Response(data=serializer.errors, status=status.HTTP_400_BAD_REQUEST)
```

Comment:
According to swagger data={"message": serializer.errors, .....} we must return message field.

Reply:
I actually talk with front team and response approved from his side
{
    "username": [
        "A user with that username already exists."
    ],
    "email": [
        "user with this email already exists."
    ]
}
Like this